module/userProjects/validator/v1.js

```diff
@@ -19,7 +19,14 @@ module.exports = (req) => {
 			req.checkQuery('taskId').exists().withMessage('required task id')
 		},
 		add: function () {
-			req.checkBody('title').exists().withMessage('required project title')
+			req.checkBody('program.name').exists().withMessage('required program name')
+			req.checkBody('program.startDate').exists().withMessage('required program start date')
+			req.checkBody('program.conversation').exists().withMessage('required program conversation')
```

Comment:
@Prajwal17Tunerlabs lets kep thi optional. they haven't confirmed passing of this data

Reply:
ok @VISHNUDAS-tunerlabs i have commented these fields for now, once the gritworks give confirmation we can work on it.